specs/IsSmartScreenRequired

```diff
@@ -0,0 +1,94 @@
+If CoreWebView2Setting.IsSmartScreenRequired is true for any CoreWebView2 in its associated CoreWebView2Environment, then SmartScreen is enabled. If CoreWebView2Setting.IsSmartScreenRequired is false for all CoreWebView2s in their CoreWebView2Environment, then SmartScreen is disabled.
+The default value for `IsSmartScreenRequired` is true if SmartScreen is enabled when the CoreWebView2 is created and false if SmartScreen is disabled when the CoreWebView2 is created. The value doesn't change if SmartScreen is enabled or disabled later.
+
+Changes to `IsSmartScreenRequired` take effect on the next navigation or download.
+
+
+# Examples
+```cpp
+wil::com_ptr<ICoreWebView2Settings> m_webViewSettings;
+void SettingsComponent::ToggleSmartScreenRequired()
+{
+    wil::com_ptr<ICoreWebView2Settings11> coreWebView2Settings11;
+    coreWebView2Settings11 =
+                m_webViewSettings.try_query<ICoreWebView2Settings11>();
+    if(coreWebView2Settings11) 
+    {
+        BOOL enabled;
+        CHECK_FAILURE(coreWebView2Settings11->get_IsSmartScreenRequired(&enabled));
+        CHECK_FAILURE(coreWebView2Settings11->put_IsSmartScreenRequired(enabled ? FALSE : TRUE));
+    }
+}
+```
+
+```c#
+void ToggleSmartScreenRequired()
+{
+    var settings = webView2Control.CoreWebView2.Settings;
+    settings.IsSmartScreenRequired = !settings.IsSmartScreenRequired;
+}
+```
+
+# Remarks
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+```cpp
+[uuid(d667d3a7-c1b7-479f-8833-db7547df6687), object, pointer_default(unique)]
+interface ICoreWebView2Settings11 : ICoreWebView2Settings10 {
+  /// SmartScreen helps you identify reported phishing and malware websites  
+  /// and also helps you make informed decisions about downloads. 
+  /// `IsSmartScreenRequired`  is used to control whether SmartScreen enabled or not. 
+  /// SmartScreen is enabled or disabled for all CoreWebView2s in a CoreWebView2Environment. 
+  /// If CoreWebView2Setting.IsSmartScreenRequired is true for any CoreWebView2 in its associated 
+  /// CoreWebView2Environment, then SmartScreen is enabled. If CoreWebView2Setting.IsSmartScreenRequired 
+  /// is false for all CoreWebView2s in the associated CoreWebView2Environment, then SmartScreen is disabled.
+  /// When it is changed, the change will be applied to all WebViews using the 
+  /// same user data folder. 
+  /// The default value for `IsSmartScreenRequired` is true if SmartScreen is enabled when the CoreWebView2 
+  /// is created and false if SmartScreen is disabled when the CoreWebView2 is created. The value doesn't change 
+  /// if SmartScreen is enabled or disabled later.
+  /// \snippet SettingsComponent.cpp ToggleSmartScreenEnabled.
```

Comment:
is this referring to something else in the GitHub repo, and if so can it be a link or full url?

Reply:
This is a doxygen statement that's a part of our tooling that produces the WebView2 COM reference documentation markdown. This should be equivalent to the COM sample code above. We do need it in the COM IDL.